var/spack/repos/builtin/packages/hypre/package.py

```diff
@@ -40,6 +41,9 @@ def install(self, spec, prefix):
         if '+shared' in self.spec:
             configure_args.append("--enable-shared")
 
+        if not self.compiler.f77 and not self.compiler.fc:
+            configure_args.append('--enable-fortran=no')
```

Comment:
Hi Todd,
This is correct
Ulrike

From: Todd Gamblin [mailto:notifications@github.com]
Sent: Thursday, March 31, 2016 1:59 PM
To: LLNL/spack <spack@noreply.github.com>
Cc: Yang, Ulrike Meier <yang11@llnl.gov>
Subject: Re: [LLNL/spack] Osx hypre (#706)


In var/spack/repos/builtin/packages/hypre/package.py<https://github.com/LLNL/spack/pull/706#discussion_r58126700>:

> @@ -40,6 +41,9 @@ def install(self, spec, prefix):

>          if '+shared' in self.spec:

>              configure_args.append("--enable-shared")

>

> +        if not self.compiler.f77 and not self.compiler.fc:

> +            configure_args.append('--enable-fortran=no')

@mathstuf<https://github.com/mathstuf>: @davydden<https://github.com/davydden>: I am not sure the Hypre guys actively maintain the CMake build system, or that it does all that the autotools one does. It was put there for a customer who used windows, and I think the team mainly updates the autotools one. So it might be good to go along with what they've got.

@tzanio<https://github.com/tzanio> @rfalgout<https://github.com/rfalgout> @ulrikeyang<https://github.com/ulrikeyang>: is this accurate?

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub<https://github.com/LLNL/spack/pull/706/files/595e15f5da6d37e088471a091513455e39dadae6#r58126700>